.changeset/fresh-fans-study.md

```diff
@@ -0,0 +1,18 @@
+---
+'@astrojs/db': minor
+---
+
+Changes how typegen works
+
+The generated dts file is now at a new location:
+
+```diff
+- .astro/db-types.d.ts
++ .astro/astro/db.d.ts
+```
+
+The update made to `src/env.d.ts` can be undone:
+
+```diff
+- /// <reference path="../.astro/db-types.d.ts" />
+```
```

Comment:
This sounds to me like I need to undo something **in this update**, but I think you mean something else by “update” right?

Something like this?

````suggestion
The following line can now be removed from `src/env.d.ts`:

```diff
- /// <reference path="../.astro/db-types.d.ts" />
```
````